apps/backend/src/routes/auth.ts

```diff
@@ -8,7 +8,7 @@ import prisma from "../db";
 import asyncHandler from "../util/asyncHandler";
 import passport from "passport";
 import { PrismaClientKnownRequestError } from "@prisma/client/runtime";
-import { refreshAccessToken } from "../auth/middleware";
+import requireAuth, { refreshAccessToken } from "../auth/middleware";
```

Comment:
This is unused

https://github.com/MTU-CS3141-Team15/CS3141-R02-team15/runs/5886830934?check_suite_focus=true#step:8:14